src/test/java/com/google/api/generator/engine/BUILD.bazel

```diff
@@ -25,3 +27,17 @@ TESTS = [
         "@junit_junit//jar",
     ],
 ) for test_name in TESTS]
+
+TEST_CLASS_NAME = "com.google.api.generator.engine.JavaCodeGeneratorTest"
+
+# Run `bazel run src/test/java/com/google/api/generator/engine:JavaCodeGeneratorTest_update`
+# to update goldens as expected generated code.
+updated_golden(
+    name = "JavaCodeGeneratorTest_update",
+    srcs = [
+        ":engine_files",
+        "//src/test/java/com/google/api/generator/engine/goldens:goldens_files",
```

Comment:
What is the motivation for including all the golden files and all the framework files?

Reply:
They are needed now, removed. Thanks